components/shared/Heading1.tsx

```diff
@@ -0,0 +1,30 @@
+type Heading1Props = {
+  children: React.ReactNode
+  dataTestId?: string
+  textColor?: string
+  textSize?: string
+  textSizeSM?: string
+  textSizeLG?: string
+  customCss?: string | null
+}
+
+const Heading1 = ({
+  children,
+  dataTestId = '',
+  textColor = 'text-violet-600',
+  textSize = '5xl',
```

Comment:
Somehow rows `15, 16, 17` are the possible origin for this bug. The fonts worked before. CSS was applied. Not after the change.

Reply:
Fixed with: https://github.com/krsiakdaniel/portfolio-website-krsiak-cz/pull/172/files